platform-services-handler/config/test-config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The configs are not in sync with internal version, Validate what features are missing.  For example:
  "SERVICE_DELETION_EVENT_START": "CALL_DELETE_WORKFLOW",
  "SERVICE_DELETION_EVENT_END" : "DELETE_PROJECT",

Reply:
At present Service deletion events feature is not included